app/bundles/LeadBundle/Event/CompanyMergeEvent.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
```

Comment:
New classes should declare strict types. Leave it as is for existing classes as adding it to an existing class could break unexpected code.
```suggestion

declare(strict_types=1);

```